cosmos-gui/conf/cosmos-gui.json

```diff
@@ -30,6 +30,10 @@
     "port": 3306,
     "user": "",
     "password": "",
-    "database": "cosmos"
+    "database": "cosmos
+  },
+  "log": {
+    "file_name": "/var/log/cosmos/cosmos-gui.log",
+    "date_pattern": ".dd-MM-yyyy"
```

Comment:
Not really sure why the previous section appears coloured this way... :O Any clue? :)